tests/config/pytest_software.yml

```
@@ -290,6 +290,10 @@ gatk4/variantfiltration:
   - software/gatk4/variantfiltration/**
   - tests/software/gatk4/variantfiltration/**
 
+gatk4_intervallisttools:
```

```suggestion
gatk4/intervallisttools:
```
that bad boy was causing the trouble